tests/test_hpack_echo.c

```diff
@@ -274,23 +263,20 @@ AWS_TEST_CASE(hpack_stress, test_hpack_stress)
 static int test_hpack_stress(struct aws_allocator *allocator, void *ctx) {
     /* Test that makes tons of streams with all sorts of headers to stress hpack */
     (void)ctx;
-    struct aws_byte_cursor host_name = aws_byte_cursor_from_c_str("httpbin.org");
+    struct aws_byte_cursor host_name = aws_byte_cursor_from_c_str("localhost");
     ASSERT_SUCCESS(s_tester_init(&s_tester, allocator, host_name));
     /* wait for connection connected */
     ASSERT_SUCCESS(s_wait_on_connection_connected(&s_tester));
-    // httpbin.org/headers is an echo server that will return the headers of your request from the body.
+    // localhost/echo is an echo server that will return the headers of your request from the body.
     struct aws_http_header request_headers_src[] = {
         DEFINE_HEADER(":method", "GET"),
         DEFINE_HEADER(":scheme", "https"),
-        DEFINE_HEADER(":path", "/headers"),
-        DEFINE_HEADER(":authority", "httpbin.org"),
+        DEFINE_HEADER(":path", "/echo"),
+        DEFINE_HEADER(":authority", "localhost"),
     };
-    /* TODO: The initail settings header table size is 4096 octets, not sure about why server response with 400 when we
-     * sent a request with around 100 headers */
-    size_t num_to_acquire = 1000;
-    size_t accpected_error = 50;
-    size_t num_headers_to_make = 70; /* will have bad request around 100 */
-    size_t error_count = 0;
+
+    size_t num_to_acquire = 5000;
```

Comment:
let's test aws_http_header_compression

maybe just run this test again, but randomly assigning compression levels